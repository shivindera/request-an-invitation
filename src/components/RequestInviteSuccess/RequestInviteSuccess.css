.broccoli-request-success {
  display: flex;
  flex-direction: column;
  align-items: center;
}

.broccoli-request-success-heaader {
  font-size: 24px;
  text-align: center;
  margin: 24px 0 16px;
}

.broccoli-request-success-border {
  width: 40px;
  margin: 0 auto 40px;
  height: 1px;
  background: black;
  display: inline-block;
}

.broccoli-request-success-content {
  font-size: 20px;
  line-height: 24px;
  margin: 0 0 40px;
  text-align: center;
}

.broccoli-request-success button {
  margin-bottom: 24px;
}
