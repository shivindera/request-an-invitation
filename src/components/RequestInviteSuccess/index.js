// The Request Invite success componet which is shown when the details are submitted successfully to the server

import { Button } from '../../utils';
import './RequestInviteSuccess.css';

const RequestInviteSuccess = (props) => {
  // Callback to when the user clicks on Ok (to close the popup)
  const { onDone } = props;

  return (
    <div className='broccoli-request-success'>
      <h2 className='broccoli-request-success-heaader'>All done!</h2>
      <span className='broccoli-request-success-border'></span>
      <p className='broccoli-request-success-content'>
        You will be one of the first to experience
        <br />
        Broccoli & Co. when we launch.
      </p>
      <Button type='primary' onClick={onDone}>
        Ok
      </Button>
    </div>
  );
};

export default RequestInviteSuccess;
