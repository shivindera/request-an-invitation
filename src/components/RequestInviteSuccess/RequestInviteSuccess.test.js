import { render } from '@testing-library/react';
import RequestInviteSuccess from './';

it('renders Request Invite Success as expected', () => {
  const requestInviteSuccess = render(<RequestInviteSuccess />);
  expect(requestInviteSuccess.container).toMatchSnapshot();
});
