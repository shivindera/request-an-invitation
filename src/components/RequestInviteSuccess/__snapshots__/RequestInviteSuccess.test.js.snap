// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Request Invite Success as expected 1`] = `
<div>
  <div
    class="broccoli-request-success"
  >
    <h2
      class="broccoli-request-success-heaader"
    >
      All done!
    </h2>
    <span
      class="broccoli-request-success-border"
    />
    <p
      class="broccoli-request-success-content"
    >
      You will be one of the first to experience
      <br />
      Broccoli & Co. when we launch.
    </p>
    <button
      class="broccoli-button broccoli-button--primary"
      type="button"
    >
      Ok
    </button>
  </div>
</div>
`;
